src/quote/QuoteSchema.ts

```diff
@@ -0,0 +1,74 @@
+const QuoteSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: 'KILT:quote:QUOTEHASH',
+  type: 'object',
+  title: 'Quote',
+  properties: {
+    attesterAddress: {
+      type: 'string',
+      title: 'The Attester ID Schema',
+      pattern: '^(.*)$',
```

Comment:
Is this necessary, if it just allows everything?

Reply:
please also check all the other patterns in this file

Reply:
No, it isn't. 

I have removed them all. Expect the one for the Terms & Conditions. It could be a URL.